src/appleseed.studio/CMakeLists.txt

```diff
@@ -644,6 +644,7 @@ target_link_libraries (appleseed.studio
     ${QT_LIBRARIES}
     ${OPENGL_LIBRARY}
     ${PYTHON_LIBRARIES}
+    uuid

```

Comment:
This will not work correctly on Windows, and maybe on OSX (I don't know if there is an uuid lib on OSX)